stock_inventory_verification_request/models/stock_inventory.py

```diff
@@ -71,3 +63,20 @@ def action_open_svr(self):
             result["views"] = [(view and view.id or False, "form")]
             result["res_id"] = request_svr_ids[0] or False
         return result
+
+    def action_request_verification(self):
+        for quant in self:
+            if quant.discrepancy_threshold and (
+                quant.discrepancy_percent > quant.discrepancy_threshold
+            ):
+                self.env["stock.slot.verification.request"].create(
+                    {
+                        "quant_id": quant.id,
+                        "location_id": quant.location_id.id,
+                        "state": "wait",
+                        "product_id": quant.product_id.id,
+                        "company_id": self.company_id.id,
+                        "lot_id": quant.lot_id.id if quant.lot_id else False,
+                    }
+                )
+                self.requested_verification = True
```

Comment:
`quant.requested_verification = True`

Reply:
And add : `return {"type": "ir.actions.act_window_close"}` which is more correct syntaxically.

Reply:
Thank you for yours comments, we have applied the changes.